README.md

```diff
@@ -1,4 +1,28 @@
-Disclaimer: suboptimal in many ways, but more than enough for what I need.
+# Branch search
+
+A simple terminal branch search for git
+
+## Install
+
+You can install branchsearch by cloning the repo:
+
+`git clone https://github.com/npazosmendez/branchsearch.git`
+
+Go into the repository's folder and running (if you want it installed automatically):
+
+`sudo make install`
+
+Otherwise you may run
+
+`sudo make`
+
+and copy the generated exectable (`bs`) to your preferred location.
+
+#### Disclaimer
+
+Suboptimal in many ways, but more than enough for what I need.
+
+## Example
 
 ![example](img.gif)
```

Comment:
I don't think this section is needed (for now). I'd leave the gif just after the project's description "A simple terminal branch search for git"

Reply:
As you wish